lib/tasks/quality.rake

```diff
@@ -39,6 +39,21 @@ namespace :quality do
     abort
   end
 
+  desc 'Runs ERB linter'
+  task :erb_linter do |task|
+    puts "########### #{task.full_comment} ###########"
+    sh 'bundle exec erblint --autocorrect app/views', verbose: false
+  end
+
+  desc 'Runs HTML linter'
+  task :html_linter do |task|
+    puts "########### #{task.full_comment} ###########"
+
+    path = Rails.root.join('node_modules', '.bin', 'htmlhint').to_s
+
+    sh "#{path} --config .htmlhintrc app/views/**/**/**/*.html.erb", verbose: false
```

Comment:
Programın bulunması işini `yarn`'a verelim (daha portable).  path ataması ve sh invoke yerine sadece aşağıdaki satır yeterli (test ettim):

```sh
sh "yarn run htmlhint --config .htmlhintrc 'app/views/**/*.html.erb'", verbose: false
```

Burada `'app/views/**/*.html.erb'` (tek tırnaklar) enteresan.  Tek tırnak vermezsen glob'u shell açıyor.  Bunun, yani tek tırnak kullanmadan çalıştırmanın bir kaç sakıncasını gözledim.  Zevkine yazayım :-)

- htmlhint her argüman için config load yapıyor.  Anladığım kadarıyla argüman listesinde main'i invoke ediyor.   Tek tırnak verildiğinde glob expand edilmeden programa gidiyor ve program tek argüman görerek glob'u kendisi açıyor.  O vakit main kodu bir kere invoke ediyor.

- `sh`  ile `/bin/sh` çalıştırılıyor.  Debian türevi sistemlerde bu ön tanımlı olarak Dash'tir ve Dash extended globları (yani `**/*`) açabilir.  Ama `/bin/sh` aslında POSIX uyumlu shell demek ve POSIX standardında bu glob desteklenmez (daha doğrusu `**` sessizce `*`'a dönüştürülür).  Dolayısıyla tek tırnaksız kullanarak glob expansion shell'e verilirse bunun her ortamda çalışacağı garanti edilemez (örneğin Dash yerine tam POSIX kullanan bir shell kurulu sistemler).  Ha ne olur, açılamayan glob programa gider ama bu bir yan etki.  Sonuç olarak glob expansion'ı tek tırnaklar yoluyla htmlhint'e vermek daha doğru.

Reply:
`yarn exec` kullanarak komitledim ve test ettim.Sizin de test etmenizde yarar var.